src/actions/genericActions.js

```diff
@@ -146,3 +147,36 @@ export function handleUpdate(data, componentName) {
     rearrange: false
   };
 }
+
+export function handleCsvShow(componentName) {
+  return {
+    type: "HANDLE_CSV_SHOW" + "--" + componentName,
+    csv: true
+  };
+}
+
+export function handleCsvHide(componentName) {
+  return {
+    type: "HANDLE_CSV_HIDE" + "--" + componentName,
+    csv: false
+  };
+}
+
+export function fetchAffordances(componentName) {
+  return (dispatch) => {
+    axios.get(urlGetAffordances(), {
+      params: {
+        model: componentName
+      }
+    })
+    .then(response => {
+      dispatch({
+        type: "FETCH_AFFORDANCES" + "--" + componentName,
```

Comment:
You can use `` `FETCH_AFFORDANCES--${componentName}` `` but don't change it here. Will break consistency but keep this in mind for later projects :smile: